src/io/input.jl

```diff
@@ -69,40 +69,102 @@ function readFlowFrame(filename::String)
 end
 
 """
-    loadData(idx, fn, md,panel; method = "asinh", cofactor = 5,
-            reduce = true, sort = true)
+    loadData(dataPath, data, nWorkers; panel=Nothing(), 
```

Comment:
is it worth typing `data`?

Reply:
yes, data could be a single data file or a meta-data file